src/core/Checkbox/index.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Icon from '../Icon'
+
+import { Label, Help } from '../helpers'
+
+import s from './styles'
+
+class Checkbox extends React.Component {
+    state = {
+        checked: !!this.props.checked,
+        indeterminate: !!this.props.indeterminate,
+    }
+
+    onClick = () => {
+        const checked = !this.state.checked
+        this.props.onChange(this.props.name, checked)
+        this.setState({ checked, indeterminate: false })
+    }
+
+    icons = {
```

Comment:
we can make them static or move them out of `class Checkbox`

Reply:
71b5ad8